.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - checkout
       - run: ./build_tools/circle/checkout_merge_commit.sh
       - restore_cache:
-          key: v1-datasets-{{ .Branch }}
+          key: doc-min-deps-datasets-{{ .Branch }}
```

Comment:
Let's make it explicit that it's natural to have to increment a counter in the cache key from time to to time.

```suggestion
          key: v1-doc-min-deps-datasets-{{ .Branch }}
```